core/src/avm1/globals/bitmap_filter.rs

```diff
@@ -53,56 +53,27 @@ pub fn clone<'gc>(
                 gradient_bevel_filter.duplicate(activation.context.gc_context),
             )
         }
-        _ => NativeObject::None,
-    };
-    if !matches!(native, NativeObject::None) {
-        let proto = this.get_local_stored("__proto__", activation);
-        let cloned = ScriptObject::new(activation.context.gc_context, None);
-        // Set `__proto__` manually since `ScriptObject::new()` doesn't support primitive prototypes.
-        // TODO: Pass `proto` to `ScriptObject::new()` once possible.
-        if let Some(proto) = proto {
-            cloned.define_value(
-                activation.context.gc_context,
-                "__proto__",
-                proto,
-                Attribute::DONT_ENUM | Attribute::DONT_DELETE,
-            );
+        NativeObject::GradientGlowFilter(gradient_glow_filter) => {
+            NativeObject::GradientBevelFilter(
```

Comment:
Shouldn't this be `GradiendGlowFilter(...)`?

Reply:
Yeah, great catch! A copy-paste mistake...